util/topgen/intermodule.py

```diff
@@ -69,6 +69,93 @@ def get_suffixes(ims: OrderedDict) -> (str, str):
     return ("_req", "_rsp")
 
 
+def add_intermodule_connection(obj, req_m, req_s, rsp_m, rsp_s):
```

Comment:
nit: should we name these suffixes `_h` and `_d` instead?

Reply:
+1

Reply:
You mean not `_h2d` `_d2h` but `_h`, `_d` for tlul signal? New implementation just does `_req` `_rsp` for TL-UL port as same manner as other intermodule signal.

Reply:
o no, i think @msfschaffner meant the input variable names, instead of `req_m`, we use `req_h`.  `rsp_d` instead of `rsp_s`

Reply:
it is "module" and "signal". Not the overloaded terms :)

Reply:
o dang, you're right.
Could you just add a comment in the definition header say that's what it is? :)
I'm sure i won't be the last to ask.

Reply:
Revised

Reply:
ahh got it. sorry for the false alarm :)